org.eclipse.wildwebdeveloper.tests/pom.xml

```diff
@@ -35,6 +35,29 @@
 					<useUIHarness>true</useUIHarness>
 				</configuration>
 			</plugin>
+			<plugin>
```

Comment:
If you commit the angular-app (which is a good thing IMO), you don't need to put the logic to dynamically fetch them at build-time in the pom.xml, do you?

Or you can remove the content files and keep this block instead.

If you decide to keep the sources checked-in (that'd be my favorite), can you please trim the application to the very minimal set of files and contents for the unit test to be relevant?

Reply:
This is still necessary to perform an npm install for angular dependencies, otherwise the tsserver will report import errors rather than template errors, which doesn't tell if the angular service is working or not. And I'll check to see if any sources/dependencies can be removed.

Reply:
Ok.
Another possibility would be to perform the `npm` step during test execution. It makes things a bit simpler to maintain, but add some extra time to test every time.

Reply:
Sure, updated the patch to do npm install at test time.